gtsam/slam/tests/testEssentialMatrixFactor.cpp

```diff
@@ -514,24 +729,81 @@ TEST (EssentialMatrixFactor3, extraTest) {
 
     // Use numerical derivatives to calculate the expected Jacobian
     Matrix Hexpected1, Hexpected2;
-    boost::function<Vector(const EssentialMatrix&, double)> f = boost::bind(
-        &EssentialMatrixFactor3::evaluateError, &factor, _1, _2, boost::none,
-        boost::none);
-    Hexpected1 = numericalDerivative21<Vector2, EssentialMatrix, double>(f, bodyE, d);
-    Hexpected2 = numericalDerivative22<Vector2, EssentialMatrix, double>(f, bodyE, d);
+    boost::function<Vector(const EssentialMatrix &, double)> f =
+        boost::bind(&EssentialMatrixFactor3::evaluateError, &factor, _1, _2,
+                    boost::none, boost::none);
+    Hexpected1 =
+        numericalDerivative21<Vector2, EssentialMatrix, double>(f, bodyE, d);
+    Hexpected2 =
+        numericalDerivative22<Vector2, EssentialMatrix, double>(f, bodyE, d);
 
     // Verify the Jacobian is correct
     EXPECT(assert_equal(Hexpected1, Hactual1, 1e-6));
     EXPECT(assert_equal(Hexpected2, Hactual2, 1e-8));
   }
 }
+/*
+TEST(EssentialMatrixFactor4, minimizationWithStrongCal3BundlerPrior2) {
```

Comment:
This test does not pass because calibrate() does not converge for our data. Should we remove it? @ayushbaid